### PR TITLE
Create SEF dummy dev page

### PR DESCRIPTION
**Pull request by RandilaP**
## Purpose

The purpose of this PR is to add a new page for SEF engineering team.

## Goals
- Introduce engineering team
- Show projects done by team.

## Preview Link
https://pr-1218-sef-site.surge.sh/

## Approach
- Add a description
- Add a projects section
- Adjust nav bar

### Screenshots
[Screencast from 2022-10-01 07:41:04.webm](https://user-images.githubusercontent.com/106733522/193379096-ca3801c3-34bd-4983-af32-941658f9a2bb.webm)

image
image
image

### CODE REVIEW

File: devpage.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                            
+                            <div class="col-md-6">
+                            <a target="_blank"  href="https://github.com/sef-global/scholarx
+                            ">
+                                <button class="btn btn-success" href="examples"><i class="fab fa-github pr-2"></i>View in github</button>
+                            </a>
+                            </div>
+                            <div class="col-md-6">
+                                <a target="_blank"  href="https://sef-scholarx.herokuapp.com/ 
+
```
Comment (jayasanka-sack) [inline devpage.html]:
```suggestion
```

File: devpage.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+                        <h3 class="mb-3">AcadeMix</h3>
+                        
+                        <p class="text-justify">"AcadeMix aims to curate a collection of free education resources available for Sri Lankan students to maximise their learning potential from Primary through to Post Graduate education and beyond."</p>
+                        <div class="row">
+                            
+                        <div class="col-md-6">
+                        <a target="_blank"  href=" https://github.com/sef-global/sef-core
+
+                        ">
+                            <button class="btn btn-success " href="examples"><i class="fab fa-github pr-2"></i>View In github</button>
```
Comment (jayasanka-sack) [inline devpage.html]:
```suggestion
                        <a target="_blank"  href=" https://github.com/sef-global/sef-core
                        ">
                            <button class="btn btn-success " href="examples"><i class="fab fa-github pr-2"></i>View In github</button>
```

File: devpage.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        </div>
+        <div class="row align-items-center justify-content-center mb-3 mb-md-5">
+            <div class="col-md-6">
+                <img src="assets/img/projects/academix.png" class="img-fluid align-self-center w-75 align-md-left hide-sm-image">
+            </div>
+            <div class="col-md-6">
+                <div class="d-flex px-3">
+                    <div class="pl-4">
+                        <h3 class="mb-3">AcadeMix</h3>
+
```
Comment (jayasanka-sack) [inline devpage.html:101]:
```suggestion
```

File: devpage.html [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+<script src="assets/js/global-custom-js.js"></script>
+<!-- Script for static navbar and footer -->
+<script src="assets/js/navigation.js"></script>
+<script src="assets/js/functions.js"></script>
+<script>
+    $(function () {
+        loadNavAndFooter('assets/content/static')
+    })
+</script>
+</html>
```
Comment (jayasanka-sack) [inline devpage.html]:
```suggestion
</html>

```

File: devpage.html [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+                            <div class="row">
+                            
+                            <div class="col-md-6">
+                            <a target="_blank"  href="https://github.com/sef-global/scholarx
+                            ">
+                                <button class="btn btn-success" href="examples"><i class="fab fa-github pr-2"></i>View in github</button>
+                            </a>
+                            </div>
+                            <div class="col-md-6">
+                                <a target="_blank"  href="https://sef-scholarx.herokuapp.com/
```
Comment (jayasanka-sack) [inline devpage.html]:
```suggestion
                                <a target="_blank" href="https://sef-scholarx.herokuapp.com/ 
```

**Comment by jayasanka-sack** [comment]
Awesome @RandilaP ! 🤩 Thanks for getting this done! 
General comments: 

- [ ] View in github should be `btn-default` button. 
- [ ] The open project should be`btn-outline-default` button. 
- [ ] Use [this icon](https://fontawesome.com/v4/icon/external-link) for the open project button 
- [ ] Reduce the space between the two buttons
- [ ] Increase the width of the project description. (maybe 4 columns for logo, 8 for the description)
- [ ] Remove unwanted spaces in the code.

**Comment by RandilaP** [comment]
Ok

**Comment by anjula-sack** [comment]
@RandilaP There's an unwanted image. can you remove it?

**Comment by anjula-sack** [comment]
@RandilaP can you find a suitable image from [freepik](https://www.freepik.com/free-photos-vectors/development). @jayasanka-sack We had a premium account, right?

**Comment by anjula-sack** [comment]
> Being part of the SEF engineering team is a lot more than making a pull request. You get first-hand experience in handling the latest cutting-edge technologies and it will be required of you to develop skills that are essential to being part of the modern industry. With our community being active through various mediums such as standup meetings, knowledge-sharing sessions and communication platform groups, you will soon be exposed to the startup culture, and the various practices and procedures involved with contributing to projects.

Here's the description @RandilaP.

**Comment by RandilaP** [comment]
@anjula-sack Thank you.

File: assets/content/static/navbar.html [hunk 6]
```diff
@@ -100,11 +100,19 @@ <h6 class="text-primary mb-md-1">Student Ambassador</h6>
                 </li>
 
                 <li class="nav-item dropdown">
-                    <a href="https://handbook.sefglobal.org" target="_blank" class="nav-link" role="button">
+                        <a href="https://handbook.sefglobal.org" target="_blank" class="nav-link" role="button">
```
Comment (jayasanka-sack) [inline assets/content/static/navbar.html]:
```suggestion
                    <a href="https://handbook.sefglobal.org" target="_blank" class="nav-link" role="button">
```

**Comment by anjula-sack** [comment]
image
can you fix the navbar? @RandilaP

**Comment by jayasanka-sack** [comment]
I think we have too many items on the navbar. We need to remove one. What can we do?

**Comment by jayasanka-sack** [comment]
One thing we should do is remove unnecessary social media icons.
We can keep only facebook, twitter and LinkedIn icons. (Other icons don't give much value, and people can't recognize them by looking at those icons. Especially slack and discourse) 

Also, we need to remove one menu item.

**Comment by jayasanka-sack** [comment]
Let's make the Team menu to -> `Teams` and make it a dropdown.
There will be two sub-menu items

1. Engineering team
2. Meet the team (That points to the teams.html page) (Someone please suggest a good name for this menu item @akshika47  @anjula-sack @Piumal1999 )

Meanwhile, @RandilaP rename `devpage.html` to `engineering-team.html`

**Comment by RandilaP** [comment]
@jayasanka-sack Okay.

**Comment by anjula-sack** [comment]
> One thing we should do is remove unnecessary social media icons. We can keep only facebook, twitter and LinkedIn icons. (Other icons don't give much value, and people can't recognize them by looking at those icons. Especially slack and discourse)
> 
> Also, we need to remove one menu item.

I think we better keep the slack at least. People know the slack icon.

**Comment by jayasanka-sack** [comment]
> I think we better keep the slack at least. People know the slack icon.

Maybe we can place them in the footer. The navbar should contain only the links users often visit or requires attention.

**Comment by anjula-sack** [comment]
> Member

yeah we can keep them in the footer

**Comment by akshika47** [comment]
Hi, @jayasanka-sack I don't think the Engineering should go as a seperate team, as everyone belongs together in the SEF team. Our Engineering 'practice' would be ideal for sitting under our programs (I rephrased it as initiatives). Let me know what you think?

image

**Comment by jayasanka-sack** [comment]
Yeah, that makes sense. I love the way you have structured the menu. 
@RandilaP Shall we update it?

**Comment by RandilaP** [comment]
@jayasanka-sack Okay.

**Comment by anjula-sack** [comment]
Any updates? @RandilaP Do you need any help?

**Comment by RandilaP** [comment]
@jayasanka-sack Can u check this

**Comment by RandilaP** [comment]
@jayasanka-sack @anjula-sack Can u review this.

**Review by anjula-sack — DISMISSED** [review]
Good job! @RandilaP 

One small change.

image
`Projects` needs to be changed to `Initiatives`

and make your PR ready for review as well

File: devpage.html [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+                        <h3 class="mb-3">ScholarX</h3>
+                        
+                        <p class="text-justify">"ScholarX is an exclusive program aimed at providing mentoring support for talented undergraduate students in Sri Lanka. Our mentors are engaged with some of the world’s top universities and Fortune 500 companies and are carefully selected for their expertise in academia and industry.
+                            This free premium mentoring platform was created for Sri Lankans who see the opportunity and positive change brought about by a culture of knowledge and expertise sharing without the limitations of geographical borders."</p>
+                            <div class="row">
+                            
+                            <div class="col-md-5">
+                            <a target="_blank" href="https://github.com/sef-global/scholarx">
+                                <button class="btn btn-default" href="examples"><i class="fab fa-github pr-2"></i>View in github</button>
+                            </a>
```
Comment (jayasanka-sack) [inline devpage.html]:
```suggestion
                            <a href="https://github.com/sef-global/scholarx" class="btn btn-default" target="_blank" >
                                <i class="fab fa-github pr-2"></i>View in github
                            </a>
```

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done, @RandilaP !

**Pull request merged**